carr/settings_shared.py

```diff
@@ -81,7 +81,20 @@
 COURSEAFFILS_EXEMPT_PATHS = ANONYMOUS_PATHS
 COURSEAFFIL_AUTO_MAP_GROUPS = ['demo']
 
-DEFAULT_SOCIALWORK_FACULTY_UNIS = []
+DEFAULT_SOCIALWORK_FACULTY_UNIS = [
+    'amo1',
+    'ym2411',
+    'srf2141',
+    'bv2204',
+    'rb3201',
+    'dls3',
+    'jkt4',
+    'vc2162',
+    'lb2106',
+    'oc2331',
+    'sc5292',
+    'ar4504'
```

Comment:
Just wanted to make sure you're aware we also have the `salt_state/carr/local_settings.py` file which overrides the `DEFAULT_SOCIALWORK_FACULTY_UNIS` list on staging and production, and also may be a better place for these unis, depending on your intentions here.

Reply:
Was not aware! Will make that change. Thank you!

Reply:
Yeah I made a change to it recently at Danny's request. Path is actually `salt_state/apps/carr/local_settings.py`